packages/editor/src/components/post-saved-state/index.js

```diff
@@ -79,7 +79,7 @@ export class PostSavedState extends Component {
 								animateClassName
 							) }
 						>
-							<Dashicon icon="cloud" />
+							<Icon icon={ cloud } />
```

Comment:
It is increasingly feeling like we landed in a good place, syntax wise, with this.